macros/base/start_print.cfg

```diff
@@ -403,14 +403,39 @@ gcode:
     {% endif %}
 
 
-[gcode_macro _MODULE_CUSTOM1]
+[gcode_macro _MODULE_EXTRUDER_HEATING_TAP]
 gcode:
-    # ---- CUSTOM Macro section
-    # this section is reserved for personal customized start actions, which can be combined with all other start actions
-    # in order to use this, create a new macro in overrides.cfg
-    # [gcode_macro _MODULE_CUSTOM1]
-    # gcode:
-    #   ## Your custom code here
+    # Heat the nozzle to tap probing temperature ontop of the purge bucket (if available)
+    # or in the middle of the bed and wait 30s to let the nozzle dilate and stabilize
+    {% set EXTRUDER_TEMP = printer["gcode_macro _USER_VARIABLES"].tap_max_probing_temp|float %}
```

Comment:
This should be changed to use the safe extruder temp instead:
```
{% set safe_extruder_temp = printer["gcode_macro _USER_VARIABLES"].safe_extruder_temp|float %}
```

Reply:
Updated variable.